lib/protein/server.ex

```diff
@@ -150,7 +150,13 @@ defmodule Protein.Server do
 
         transport_server_opts = Keyword.put(transport_opts, :server_mod, __MODULE__)
 
-        [{transport_server_mod, transport_server_opts}]
+        [
+          %{
+            id: transport_server_mod,
+            start: {transport_server_mod, :start_link, [transport_server_opts]},
+            shutdown: 15_000
```

Comment:
There's a problem with this latest change ie. with setting this to fixed value - if processing timeout was set to more than 15s then this will not give RPC processing procs to finish & kill them. Instead we should set here the value of configured timeout + some extra time like 1s to ensure process is indeed done. Or just get rid of this and leave this edge case to sigkill.

Reply:
I think this should be independently configurable because shutdown timeout may be lower than the response timeout. What do you think about that?

Reply:
shutdown has been removed